### PR TITLE
[IOTDB-1455] Update documents for new IoTDB concepts and TsFile structures

### PR DESCRIPTION
**Pull request by samperson1997**
We have some new names for concepts and structures in IoTDB and TsFile:

For IoTDB: 
- Entity (device), measurement, timeseries
- Measurement template (device template or schema template)

For TsFile:
- ChunkIndex (ChunkMetadata), TimeseriesIndex (TimeseriesMetadata), IndexOfTimeseriesIndex (TsFileMetadata)
- Using "index area" instead of "metadata" in tsfile

This PR updates documents in:
- User Guide 3.1 Data Concept
- System Design 2.2 TsFile Format

### CODE REVIEW

File: docs/SystemDesign/TsFile/Format.md [hunk 1]
```diff
@@ -66,49 +66,60 @@
 
 ### 1.2 TsFile Overview
 
+<!-- TODO
+
 Here is the structure diagram of TsFile.
 
 <img style="width:100%; max-width:800px; max-height:600px; margin-left:auto; margin-right:auto; display:block;" src="https://user-images.githubusercontent.com/33376433/123052025-f47aab80-d434-11eb-94c2-9b75429e5c54.png">
```
Comment (samperson1997) [inline docs/SystemDesign/TsFile/Format.md:73]: This figure could be updated by @THUMarkLau later : )

Comment (samperson1997) [inline docs/SystemDesign/TsFile/Format.md:73]: This figure could be updated by @THUMarkLau later : )

**Comment by SilverNarcissus** [comment]
Nice work!

**Comment by SilverNarcissus** [comment]
Nice work!

**Review by SilverNarcissus — APPROVED** [review]
Nice work!

**Pull request merged**